src/main/java/cd/rr/calculator/Calculator.java

```diff
@@ -1,7 +1,115 @@
 package cd.rr.calculator;
 
+import java.util.Scanner;
+
+
 public class Calculator {
+
+
+    public static void Menu(){
+        System.out.println("Welcome to the CLI Calculator");
+        System.out.println("-----------------------------\n");
+        System.out.println("Opperations: ");
+        System.out.println("""
+                1: Addition
+                2: Subtraction
+                3: Multiplication
+                4: Division
+                5: Exit""");
+        System.out.print("Please choose an operation (1-5): ");
+    }
+
+    public static void addition(int num1, int num2){
+        System.out.println("Result: " + (num1 + num2));
+    }
+
+    public static void subtraction(int num1, int num2){
+        System.out.println("Result: " + (num1 - num2));
+    }
+
+    public static void multiplication(int num1, int num2){
+        System.out.println("Result: " + (num1 * num2));
+    }
+
+    public static void division(int num1, int num2){
+        double divisionResult = (double) num1 / num2;
+        double roundDivision = Math.round(divisionResult * 100.0) / 100.0;
+        System.out.println("Result: " + (roundDivision));
+    }
     public static void main(String[] args) {
+        int inNum1;
+        int inNum2;
+        Scanner input = new Scanner(System.in);
+        Calculator.Menu();
+        int userInput = input.nextInt();
+        do {
+            switch (userInput) {
```

Comment:
The cases seem repetitive, perhaps pulling the prompts for each number, the menu, and the prompts for the operation outside of the switch but still in the loop?